app/experimenter/normandy/tasks.py

```diff
@@ -163,6 +165,21 @@ def update_population_percent(experiment, recipe_data):
             experiment.save()
 
 
+def update_is_high_population(experiment, recipe_data):
+    recipe_args = recipe_data.get("arguments", {})
+    if recipe_data and recipe_is_high_population := recipe_args.get("isHighPopulation"):
```

Comment:
`if recipe_args and recipe_is_high_population := recipe_args.get("isHighPopulation"):`

`recipe_args` and not `recipe_data`?

Reply:
Actually, it's kind of a superfluous check since if it defaults to `{}` and then you walrus assign from that it'll be `None` which will fail the check anyways so you should be able to just do 

`if recipe_is_high_population := recipe_args.get("isHighPopulation"):`?